### PR TITLE
Implement organizations list

### PR DESCRIPTION
**Pull request by selankon**
It implements organization list screen. Depends on #2 

image

### CODE REVIEW

File: src/components/Organizations/OrganizationsList.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  const searchOnChange = (event: any) => {
+    debouncedSearch(event.target.value)
+  }
+
+  return <InputSearch maxW={'300px'} placeholder={t('organizations.search_by_org_id')} onChange={searchOnChange} />
+}
+
+export const OrganizationsList = () => {
+  const { page, orgId }: { page?: number; orgId?: string } = useParams()
+  const { data: orgsCount, isLoading: isLoadingCount, error: countError } = useOrganizationCount()
```
Comment (elboletaire) [inline src/components/Organizations/OrganizationsList.tsx]: `countError` isn't actually being used

File: src/components/Organizations/OrganizationsList.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  }, 1000)
+
+  const searchOnChange = (event: any) => {
+    debouncedSearch(event.target.value)
+  }
+
+  return <InputSearch maxW={'300px'} placeholder={t('organizations.search_by_org_id')} onChange={searchOnChange} />
+}
+
+export const OrganizationsList = () => {
```
Comment (elboletaire) [inline src/components/Organizations/OrganizationsList.tsx]: Better call it `PaginatedOrganizationsList`. 

I can imagine multiple scenarios where we may want to just show some items listed, but with no pagination (i.e. a "last organizations" block), and have a different `OrganizationsList` component for that.

File: src/components/Footer.tsx [hunk 3]
```diff
@@ -38,7 +38,7 @@ export const Footer = () => {
     <Flex h={'90px'} mt={'auto'} bottom={0} w={'full'} align={'center'} justify={'space-between'}>
       <Box m={{ base: '20px auto', md: '0 40px' }}>
         <Link href={'/'}>
-          <Img maxH={'35px'} src='images/logo-classic.svg' alt='Vocdoni' />
+          <Img maxH={'35px'} src='/images/logo-classic.svg' alt='Vocdoni' />
```
Comment (elboletaire) [inline src/components/Footer.tsx]: Assets should be imported via `import`, i.e.

~~~ts
import logo from '/images/logo-classic.svg'
~~~

Then use it as the source:

~~~ts
<Img maxH={'35px'} src={logo} alt='Vocdoni' />
~~~

File: src/components/Organizations/Card.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+const OrganizationCardContent = ({ id, electionCount }: IOrganizationCardProps) => {
+  const { organization, loading } = useOrganization()
+  const { t } = useTranslation()
+
+  return (
+    <Card direction={'row'} alignItems='center' overflow={'scroll'} pl={4}>
+      <Box w={'50px'}>
+        <Avatar
+          mx='auto'
+          fallbackSrc={'/images/fallback-account-dark.png'}
```
Comment (elboletaire) [inline src/components/Organizations/Card.tsx:29]: I think you wanted to use `OrganizationImage`, not Avatar. Avatar does not have the fallback feature.

Edit: also use `import` for the assets, as stated above.

File: src/components/Organizations/OrganizationsList.tsx [hunk 5]
```diff
@@ -0,0 +1,70 @@
+import { InputSearch } from '~src/layout/Inputs'
+import { useOrganizationCount, useOrganizationList } from '~queries/organizations'
+import { debounce } from '~utils/debounce'
+import { generatePath, useNavigate, useParams } from 'react-router-dom'
+import { RoutedPaginationProvider } from '~components/Pagination/PaginationProvider'
+import { Loading } from '~src/router/SuspenseLoader'
```
Comment (elboletaire) [inline src/components/Organizations/OrganizationsList.tsx]: Move the `Loading` component to `src/layout/Loading.tsx`. Kinda confusing to have it in the SuspenseLoader, considering it'll be used in many places.

Also, the import order doesn't seem correct. Have you enabled auto-sort for this project? 😄

File: src/components/Organizations/OrganizationsList.tsx [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+      return generatePath('/organizations/:page?/:orgId?', { page: page?.toString() || '0', orgId: value as string })
+    }
+    navigate(getPath())
+  }, 1000)
+
+  const searchOnChange = (event: any) => {
+    debouncedSearch(event.target.value)
+  }
+
+  return <InputSearch maxW={'300px'} placeholder={t('organizations.search_by_org_id')} onChange={searchOnChange} />
```
Comment (elboletaire) [inline src/components/Organizations/OrganizationsList.tsx:25]: Avoid typing with `any`. Also, no need to create a function for this... just place it on the component:

```suggestion
  return (
    <InputSearch
      maxW={'300px'}
      placeholder={t('organizations.search_by_org_id')}
      onChange={(event: ChangeEvent<HTMLInputElement>) => {
        debouncedSearch(event.target.value)
      }}
    />
  )
```

File: src/components/Organizations/OrganizationsList.tsx [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+  const navigate = useNavigate()
+
+  const debouncedSearch = debounce((value) => {
+    const getPath = () => {
+      // If previous state has not org id, ensure to look at the first page
+      if (!orgId || orgId.length === 0) {
+        return generatePath('/organizations/:page?/:orgId?', { page: '0', orgId: value as string })
+      }
+      return generatePath('/organizations/:page?/:orgId?', { page: page?.toString() || '0', orgId: value as string })
+    }
```
Comment (elboletaire) [inline src/components/Organizations/OrganizationsList.tsx]: I feel like here we're overcomplicating things, but I'm not sure what would be the proper solution. I'd leave it simple for now, and we'll see what we do with the case of start searching when not on the first page.

```suggestion
    const getPath = () => generatePath('/organizations/:page?/:orgId?', { page: page || 0, orgId })
```

**Review by elboletaire — CHANGES_REQUESTED** [review]

File: src/components/Organizations/OrganizationsList.tsx (hunk 1 above)
Comment (selankon) [reply src/components/Organizations/OrganizationsList.tsx]: 4a4a3ee

File: src/components/Organizations/Card.tsx (hunk 4 above)
Comment (selankon) [reply src/components/Organizations/Card.tsx:29]: 301d9c7

Comment (selankon) [reply src/components/Organizations/Card.tsx:29]: 5bfc35f

File: src/components/Footer.tsx (hunk 3 above)
Comment (selankon) [reply src/components/Footer.tsx]: 5bfc35f

File: src/components/Organizations/OrganizationsList.tsx (hunk 2 above)
Comment (selankon) [reply src/components/Organizations/OrganizationsList.tsx]: 9a7de73

File: src/components/Organizations/OrganizationsList.tsx (hunk 5 above)
Comment (selankon) [reply src/components/Organizations/OrganizationsList.tsx]: I don't have the import sorting enabled. I will add a new PR with a massive lint fix.

btw 2c3685b

File: src/components/Organizations/OrganizationsList.tsx (hunk 7 above)
Comment (selankon) [reply src/components/Organizations/OrganizationsList.tsx]: 990c189

**Review by elboletaire — APPROVED** [review]

**Pull request merged**